src/inc/til/static_map.h

```diff
@@ -17,7 +17,7 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
         using const_iterator = typename std::array<std::pair<K, V>, N>::const_iterator;
 
         template<typename... Args>
-        constexpr explicit static_map(const Args&... args) :
+        constexpr explicit static_map(const Args&... args) noexcept :

```

Comment:
I'm getting `warning C26455: Default constructor may not throw. Declare it 'noexcept'` for some reason.